test/integration/main_test.go

```diff
@@ -52,6 +53,11 @@ const (
 	testWorkflowRunAttempt         = "1"
 )
 
+const (
```

Comment:
Discussed offline. Using the index is error prone and is difficult to understand. Need to get rid of it.